src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -273,11 +273,12 @@ trait AttributeComponent {
       }
 
       def dropAttributeTempTableAction() = {
-        sql"""drop table #${baseTableRow.tableName}_TEMP""".as[Int]
+        sql"""drop temporary table if exists #${baseTableRow.tableName}_TEMP""".as[Int]
       }
 
       def upsertAction(ownerIds: Seq[OWNER_ID], insertFunction: () => ReadWriteAction[Unit]) = {
-        createAttributeTempTableAction() andThen
+        dropAttributeTempTableAction() andThen
```

Comment:
the old pattern was: 

create table if not exists
[do stuff here]
truncate table

now it's:

drop table if exists
create table
[do stuff here]
drop table if exists

there was some discussion about wanting to log when we have to do the leading drop (because something must have errored at another time to leave the temp table around). this would be great but i can't think of a way to do this. i tried a few approaches:

1) `drop if exists` works (and that's what i'm using), but there's no way to know that it actually dropped a table. the response is just the number of rows affected which always seems to be 0 since it's a whole table drop. in mysql workbench it WILL give a warning if the table didn't exist, but i have no idea how to gather this warning in code

2) `drop temporary table` will return an error if it doesn't exist and succeed if it does exist, but unfortunately the error causes the entire txn to abort so this isn't very useful. for this same reason i can't do a select or anything to determine existence 

3) you can't infer existence using information_schema because temp tables don't show up there because they aren't technically a part of the database

if we want this- does anybody have any ideas?